domain/actionSpace/urgentWishForChildren.unit.ts

```diff
@@ -6,17 +6,18 @@ import {
     Dwarf,
     EntityMutation,
     FurnishingId,
+    Game,
     isMutationOfType,
     Mutation,
     Player,
 } from "../entity";
 
 describe("Urgent Wish for Children", () => {
-    shouldPlaceDwarf(UrgentWishForChildren.execute);
+    shouldPlaceDwarf(UrgentWishForChildren.execute, initPlayerFonds);
```

Comment:
What is "fonds"? Is it Resources?

Reply:
Resources is indeed better 👍 changed https://github.com/BoredGamesWorkshop/caverna/pull/9/commits/685c56ba97ba67d5fcd7aa75c4adfc0595a4807b